client/main.py

```diff
@@ -81,10 +88,6 @@ def play():
     
     # end game logic
 
-def menu():
-    # draw the main menu here
-    # send a request to join the game
-    return False
 
 def main():
```

Comment:
Keep in mind, this function is what's running. We do not need to call `play` from a dependency.